com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,302 @@
+{
+    "app-id": "com.ozmartians.VidCutter",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "vidcutter",
+    "rename-appdata-file": "com.ozmartians.vidcutter.appdata.xml",
+    "rename-desktop-file": "com.ozmartians.vidcutter.desktop",
+    "rename-icon": "vidcutter",
+    "finish-args": [
+        "--device=dri",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=LC_NUMERIC=C",
+        "--env=QT_PLUGIN_PATH=/app/lib/qt/plugins",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--talk-name=ca.desrt.dconf",
+        "--talk-name=com.canonical.Unity.LauncherEntry"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/lib/pkgconfig",
+        "/share/man"
+    ],
+    "modules": [
+    {
+        "name": "PyOpenGL",
+        "buildsystem": "simple",
+        "build-commands": [
+            "python3 setup.py install --root=${FLATPAK_DEST} --optimize=1",
```

Comment:
This is wrong for all of these, you should be setting the prefix not root. That way you avoid the sill `mv` dance below which very likely can cause breakage.